doc/script_commands.txt

```diff
@@ -7381,9 +7393,12 @@ Suggest to use 'unitblockmove' to forcefully stop the unit with OnTouch.
 
 ---------------------------------------
 
-*unittalk <GID>,"<text>";
+*unittalk <GID>,"<text>"{,flag};
 
 This command will make a <GID> say a message. The display name of the <GID> won't get appended in front of the message.
+flag: Specify target
```

Comment:
why just don't add a flags like: `- ALL / - SELF`, i know, it's easier to modify NPC to use numeric parameter, but it's unreadable in scripts. 

Just suggestion:
`unitwalk getcharid(3), "My Super Message",area;`
or
`unitwalk getcharid(3), "My Super Message",self;`

or like with npctalk:
`unitwalk getcharid(3), "My Super Message",bc_area;`
or
`unitwalk getcharid(3), "My Super Message",bc_self;`